src/components/elements/CuratePublications/CuratePublications.tsx

```diff
@@ -27,15 +27,24 @@ const filtersList = [
 
 const CuratePublications = () => {
   const dispatch = useDispatch()
+  
+
+
   const filters = useSelector((state: RootStateOrAny) => state.filters)
-  const filteredIds = useSelector((state: RootStateOrAny) => state.identityAllData.map((identity) => identity.personIdentifier))
+  
+  const filterPidData = useSelector((state: RootStateOrAny) => state.curateIdsFromSearchPage.filter((identity) => identity.countPendingArticles > 0 ));
```

Comment:
If we have to filter `curateIdsFromSearchPage` again here to check if they have count pending or not, why aren't we just using all list of ids and filtering them instead of storing a new state?

Reply:
We are only picking ids displayed on the page selected in search results itself not all PMIDs being returned by the search. hence, storing them in a different state to use them to fetch corresponding articles in an incremental manner like 10 per page.